lib/utils/slices/slices.go

```diff
@@ -18,21 +18,20 @@
 
 package slices
 
-import (
-	"cmp"
-	"slices"
-)
-
 // FilterMapUnique applies a function to all elements of a slice and collects them.
 // The function returns the value to collect and whether the current element should be included.
 // Returned values are sorted and deduplicated.
```

Comment:
(if we do we obviously should sort the slice before returning, even if we need to use a map to deduplicate a `comparable` rather than a `cmp.Ordered`)

Reply:
We don't actually require them to be sorted, and after this PR we no longer ensure it is sorted.
Removed that as well from the godoc.
Thank you